server/settings.go

```diff
@@ -49,3 +52,34 @@ func (p *Plugin) settingsNotifications(header *model.CommandArgs, instanceID, ma
 
 	return p.responsef(header, "Settings updated. Notifications %s.", notifications)
 }
+
+func (p *Plugin) settingsWatching(header *model.CommandArgs, instanceID, mattermostUserID types.ID, connection *Connection, args []string) *model.CommandResponse {
+	const helpText = "`/jira settings watching [value]`\\n* Invalid value. Accepted values are: `on` or `off`."
+
+	if len(args) != 2 {
+		return p.responsef(header, helpText)
+	}
+
+	var value bool
+	notifications := settingOff
```

Comment:
1/5 I think these variable names seem better

```suggestion
	watching := false
	watchingOnOff := settingOff
```